process/sync/metablock.go

```diff
@@ -114,6 +119,7 @@ func NewMetaBootstrap(
 	}
 
 	base.hdrRes = hdrRes
+	base.forkInfo = process.NewForkInfo()
```

Comment:
no need for NewForkInfo function.

Reply:
I think it looks more cleaner like this process.NewForkInfo() instead of &process.ForkInfo{IsDetected: false, Nonce: math.MaxUint64, Round: math.MaxUint64, Hash: nil}, as it is used in over 28 places (including tests)